climada/hazard/test/test_relative_cropyield.py

```diff
@@ -48,19 +56,23 @@ def test_load_EU_all(self):
 
     def test_set_rel_yield(self):
         """Test setting intensity to relativ yield"""
-        haz = CropPotential()
+        haz = RelativeCropyield()
         haz.set_from_single_run(input_dir=INPUT_DIR, yearrange=(2001, 2005), ag_model='lpjml',
                                 cl_model='ipsl-cm5a-lr', scenario='historical', soc='2005soc',
                                 co2='co2', crop='whe', irr='noirr', fn_str_var=FN_STR_DEMO)
         hist_mean = haz.calc_mean(np.array([2001, 2005]))
+
+        self.assertEqual(haz.intensity_def, 'Yearly Yield')
         haz.set_rel_yield_to_int(hist_mean)
+        self.assertEqual(haz.intensity_def, 'Relative Yield')
+
         self.assertAlmostEqual(np.max(hist_mean), 8.098413, places=5)
         self.assertEqual(haz.intensity.shape, (5, 1092))
-        self.assertAlmostEqual(np.nanmax(haz.intensity.toarray()), 4.0, places=5)
-        self.assertAlmostEqual(haz.intensity.max, 4.0, places=5)
-        self.assertAlmostEqual(haz.intensity.min, 0.0, places=5)
+        self.assertAlmostEqual(np.nanmax(haz.intensity.toarray()), 3.0, places=5)
+        self.assertAlmostEqual(haz.intensity.max(), 3.0, places=5)
+        self.assertAlmostEqual(haz.intensity.min(), -1.0, places=5)
 
```

Comment:
Add tests for all the methods in the class RelativeCropyiel (except maybe the plot methods)